src/linux/helpers/nolibc-tweaks.patch

```diff
@@ -275,7 +287,7 @@ index fffdaf6ff..78eae91d0 100644
 +static void *memmove(void *dst, const void *src, size_t len)
  {
  	size_t dir, pos;
- 
+
```

Comment:
This file also has a bunch of whitespace changes that shouldn't be part of this.